xbmc/storage/discs/DiscDriveHandlerBase.h

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "IDiscDriveHandler.h"
+
+#include <string>
+
+/*! \brief Implementation of a generic/base DiscDriveHandler
+* \sa IDiscDriveHandler
+*/
+class CDiscDriveHandlerBase : public IDiscDriveHandler
```

Comment:
What is the point of this class? It seems to only be used by `CDiscDriveHandlerPosix`.

Reply:
To group any methods related to discdrivehandling common to all. I agree it's not of any use now if only used by the POSIX path however I am still not sure if stuff will have to come here at some point while working on the the other stuff. If you don't mind I'd prefer to keep this for now and come back to it later on as we go.

Reply:
It's just dead code really. You should add it when needed. It will make it cleaner and make future improvements easier to understand also.